docs/api/routing.rst

```diff
@@ -374,6 +374,54 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other architectures. Consider the task of building an API 
+for a calculator which can both add add subtract two numbers. You could implement the following:
+
+.. code:: python
+
+    class Add(object):
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) + int(req.get_param('y')))
+            resp.status = falcon.HTTP_200
+
+    class Subtract(object):
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) - int(req.get_param('y')))
+            resp.status = falcon.HTTP_200
+
+    add = Add()
+    subtract = Subtract()
+    api = falcon.API()
+    api.add_route('/add', add)
+    api.add_route('/subtract', subtract)
+
+However, in addition to being slightly verbose it's also logically a bit strange because adding and subtracting don't seem to conceptually map to two seperate resource 
+collections and they don't seem like they would ever require other HTTP verbs such as POST, DELETE, etc.
```

Comment:
I think "they don't seem like they would ever require other HTTP verbs such as POST, DELETE, etc." could be removed or reworded, since you can have multiple methods also with suffixes, and this seems to imply that would not be possible.

Reply:
Just removed and reworded a bit, let me know what you think